.gitignore

```diff
@@ -8,9 +8,9 @@ venv.vagrant
 Pipfile*
 
 # Django
-db.sqlite3
-media/
-static/
+/transport_nantes/db.sqlite3*
+/transport_nantes/media/
+/transport_nantes/static/
```

Comment:
We don't have these paths for media and static, they are transport_nantes/<app_name>/static

Reply:
Exactly, and we very much want content in `transport_nantes/*/static` to  not be ignored.

What we want ignored is the stuff that collectstatic accumulates in `transport_nantes/static/` or that we test upload to `transport_nantes/media`.

Reply:
Hmm, I see, my statics don't go in those but maybe that's my local settings. LGTM then !